src/tests/gtests.cpp

```diff
@@ -174,3 +174,74 @@ TEST_F(LeelaTest, MoveOnOccupiedSq) {
     // Find this error in the output
     EXPECT_NE(output.find("illegal move"), std::string::npos);
 }
+
+
+TEST_F(LeelaTest, TimeControl) {
+    // Initialize network
+    cfg_weightsfile = "../src/tests/0k.txt";
```

Comment:
I think you could make a Test Fixture for this, other tests will surely find it useful too.

Reply:
Moved to init_global_objects.

Reply:
Actually moved to LeelaEnv.